diff --git a/Makefile b/Makefile
index 3db2ec9..d24b879 100644
--- a/Makefile
+++ b/Makefile
@@ -25,18 +25,18 @@ all: _sass.so
 
 build2/libsass/c/%.o: libsass/src/%.c
 	@mkdir -p build2/libsass/c/
-	gcc $(CFLAGS) -c $^ -o $@
+	$(CC) $(CFLAGS) -c $^ -o $@
 
 build2/libsass/cpp/%.o: libsass/src/%.cpp
 	@mkdir -p build2/libsass/cpp/
-	gcc $(CPPFLAGS) -c $^ -o $@
+	$(CC) $(CPPFLAGS) -c $^ -o $@
 
 build2/pysass.o: pysass.cpp
 	@mkdir -p build2
-	gcc $(CPPFLAGS) -Wno-write-strings -c $^ -o $@
+	$(CC) $(CPPFLAGS) -Wno-write-strings -c $^ -o $@
 
 _sass.so: $(C_OBJECTS) $(CPP_OBJECTS) build2/pysass.o
-	g++ $(LFLAGS) $^ -o $@
+	$(CXX) $(LFLAGS) $^ -o $@
 
 .PHONY: clean
 clean:
diff --git a/setup.py b/setup.py
index a5734f0..bc356c4 100644
--- a/setup.py
+++ b/setup.py
@@ -25,8 +25,6 @@ def _maybe_clang(flags):
     if platform.system() not in ('Darwin', 'FreeBSD'):
         return
 
-    os.environ.setdefault('CC', 'clang')
-    os.environ.setdefault('CXX', 'clang++')
     orig_customize_compiler = distutils.sysconfig.customize_compiler
 
     def customize_compiler(compiler):
